aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1256,14 +1256,64 @@ static inline Tensor& bmm_out_or_baddbmm_(Tensor& self_or_result, const Tensor&
             && self_or_result.is_contiguous()) {
     at::native::_baddbmm_mkl_(self_or_result, batch1, batch2, beta, alpha);
   } else { // split along batch dimension
+    bool enable_multithreaded_bmm{false};
+#ifdef C10_MOBILE
+    /*
+     * Inference mode multithreading is done because various thread local
```

Comment:
Not sure what you mean. It already clarifies why it is enabled in inference mode. Or do you mean to capture the fact that having autograd engine running in multi-threading can be an issue?

Reply:
It is more that "Inference mode multithreading is done because" is not clear to me at first read. I would replace it with "We only do multithreading when Inference Mode is enabled because".